proto/message_contents/invitation.proto

```diff
@@ -11,6 +11,73 @@ import "message_contents/public_key.proto";
 option go_package = "github.com/xmtp/proto/v3/go/message_contents";
 option java_package = "org.xmtp.proto.message.contents";
 
+// Unsealed invitation V2
+message InvitationV2 {
+    // Supported encryption schemes
+    // AES256-GCM-HKDF-SHA256
+    message Aes256gcmHkdfsha256 {
+        bytes key_material = 1; // randomly generated key material (32 bytes)
+    }
+
+    // The context type
+    message Context {
+        // Expected to be a URI (ie xmtp.org/convo1)
+        string conversation_id = 1;
+        // Key value map of additional metadata that would be exposed to
+        // application developers and could be used for filtering
+        map<string, string> metadata = 2;
+    }
+    // topic name chosen for this conversation.
+    // It MUST be randomly generated bytes (length >= 32),
+    // then base64 encoded without padding
+    string topic = 1;
+    
+    // A context object defining metadata
+    Context context = 2;
+
+    // message encryption scheme and keys for this conversation.
+    oneof encryption {
+        // Specify the encryption method to process the key material properly.
+        Aes256gcmHkdfsha256 aes256_gcm_hkdf_sha256 = 3;
+    }
+
+    // The address of the person being invited to the conversation. This
+    // field must only be used if the send_key_bundle attached
+    // to the invitation header originates from your own account.
+    string recipient_address_untrusted = 4;
```

Comment:
As I read this, this field is encrypted on the network so it is private, and unsigned so it is deniable. Is this true?